publisher/start.sh

```diff
@@ -34,7 +34,27 @@ EORESTORE
       pg_restore /dumps/db.backup -U postgres -d ${DB_NAME}
 EORESTORE
 
+    # we need to sanitize sensitive data first
+    docker run --name sanitizing_db_container -d --rm -v ${DUMPS_DIR}:/files \
+    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= postgres:10.5
+    # wait for "database system is ready to accept connections", hoping 4 sec is enough
+    sleep 4;
```

Comment:
Please use something like this: https://github.com/plyo/postgres/blob/40c042c6780ccbaa79b2936f5222cb1afcd1787c/.circleci/config.yml#L36-L45